### ISSUE TITLE
Can't create Module

### ISSUE THREAD

**Issue by ChristianGiupponi** [AsgardCms/Platform#55]
Tried to generate a new Module but it get stuck on the Entity

~~~
vagrant@homestead:/vagrant/www/AsgardCMS$ php artisan asgard:module:scaffold
 Please enter the module name. Example: vendor/name.:
 > christian/test

 Do you want to use Eloquent or Doctrine ? [eloquent] [Eloquent]:
  [0] Eloquent
  [1] Doctrine
 > 0

 Enter entity name. Leaving option empty will continue script.:
 > Test

 Enter entity name. Leaving option empty will continue script.:
 > 

                                                                                                                        
 [ERROR] A value is required.                                                                                           
                                                                                                                        

 Enter entity name. Leaving option empty will continue script.:
 > 
~~~

**Comment by nWidart** [comment]
Hello,

This issue has been resolved in version [1.0.2](https://github.com/AsgardCms/Workshop/releases). Please update to the latest version. You can update your modules using `composer update`.

Thanks.

**Comment by ChristianGiupponi** [comment]
Thanks! I didn't notice I was using an old version!
